components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -184,6 +185,26 @@ public void addOAuthApplication(OAuthAppDO consumerAppDO) throws IdentityOAuthAd
         }
     }
 
+    private void addOAuthAppCallbackUrls(Connection connection, int appId, int tenantId, List<String> callbackUrls)
+            throws SQLException {
+
+        if (callbackUrls != null && !callbackUrls.isEmpty()) {
```

Comment:
we can use arrautils is not empty method

Reply:
Updated